integration_test/testutil_test.go

```diff
@@ -37,8 +37,10 @@ import (
 const (
 	persistentIndexCache = "krew-persistent-index-cache"
 	krewBinaryEnv        = "KREW_BINARY"
-	validPlugin          = "konfig" // a plugin in central index with small size
-	validPlugin2         = "mtail"  // a plugin in central index with small size
+	validPlugin          = "konfig"          // a plugin in central index with small size
+	validPlugin2         = "mtail"           // a plugin in central index with small size
+	localServerURL       = "127.0.0.1:45076" // change port number incase of bind error
```

Comment:
Please do not hardcode. 

httputil can start a server at a random port. 
Just return its base url and close function from test util func.

Reply:
Done